pkg/checks/external/main.go

```diff
@@ -613,7 +613,7 @@ func (ext *Checker) waitForPodRunning() chan error {
 				ext.log("pod state is now:", string(p.Status.Phase))
 				if p.Status.Phase == apiv1.PodRunning || p.Status.Phase == apiv1.PodFailed || p.Status.Phase == apiv1.PodSucceeded {
 					ext.log("pod is now either running, failed, or succeeded")
-					outChan <- nil
+					outChan <- errors.New(p.Status.Message)
```

Comment:
This function (`waitForPodRunning()`) is actually supposed to not return an error once the pod stops running - for any reason.  An error here would mean that we failed to communicate with the Kubernetes API, or had some other internal failure.